impeller/entity/geometry/fill_path_geometry.cc

```diff
@@ -147,4 +149,16 @@ std::optional<Rect> FillPathGeometry::GetCoverage(
   return path_.GetTransformedBoundingBox(transform);
 }
 
+bool FillPathGeometry::CoversArea(const Matrix& transform,
```

Comment:
I would add a few sanity unit tests that cover the rrect cases, i.e. large rect, empty rect, et cetera.

Reply:
Good idea, done.